src/matroskavideo.cpp

```diff
@@ -0,0 +1,763 @@
+// ***************************************************************** -*- C++ -*-
+/*
+ * Copyright (C) 2004-2021 Exiv2 authors
+ * This program is part of the Exiv2 distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, 5th Floor, Boston, MA 02110-1301 USA.
+ */
+// *****************************************************************************
+// included header files
+#include "config.h"
+
+//#ifdef EXV_ENABLE_VIDEO
+#include "matroskavideo.hpp"
+#include "error.hpp"
+#include "futils.hpp"
+#include "basicio.hpp"
+#include "tags.hpp"
+#include "tags_int.hpp"
+
+// + standard includes
+#include <cmath>
+#include <cassert>
+#include<iostream>
+
+// *****************************************************************************
+// class member definitions
+namespace Exiv2 {
+    namespace Internal {
+
+    //! List of composite tags. They are skipped and the child tags are read immediately
+    uint64_t compositeTagsList[] = {
```

Comment:
Should be a constexpr std::array. Same with the below one.

Reply:
done here https://github.com/Exiv2/exiv2/pull/2370/commits/65a01dc171f4dc230eed22754990d4c25dd0078d